types/styled-system/index.d.ts

```diff
@@ -7,6 +7,7 @@
 //                 Eloy Durán <https://github.com/alloy>
 //                 Matthieu Vachon <https://github.com/maoueh>
 //                 Adam Lavin <https://github.com/lavoaster>
+//                 Joachim Schuler <https://github.com/jschuler>
```

Comment:
Just want to check if the implication of adding to this header is clear? I.e. you’re signing up as a maintainer if the types and will get pinged for each new issue/PR for these types.

Reply:
Yes thanks, I'd like to help maintain these types

Reply:
Ace 🙏